### PR TITLE
Select input text when opening Quick Navigation

### PR DESCRIPTION
**Pull request by sofiaromorales**
Bug/issue #490, if applicable: 

## Summary

With this implementation, the input text is automatically selected to overwrite it with a new query when reopening Quick Navigation after a previous search.

## Dependencies

N/A

## Testing

_Describe how a reviewer can test the functionality of your PR. Provide test content to test with if
applicable._

Steps:
1. Turn on the feature flag for Quick Navigation
2. Open the Quick Navigation search bar, type a query, and proceed to close it again
3. Open Quick Navigation and assert that the input text is selected

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [X] Added tests
- [X] Ran `npm test`, and it succeeded
- [ ] Updated documentation if necessary (N/A)

### CODE REVIEW

**Comment by marinaaisa** [comment]
Thanks for submitting this PR, @sofiaromorales  👏 
I'll be OOO tomorrow so I'll review it next week.

File: tests/unit/components/Filter/FilterInput.spec.js [hunk 1]
```diff
@@ -222,6 +222,22 @@ describe('FilterInput', () => {
...
+    wrapper = shallowMount(FilterInput, {
+      propsData: {
+        ...propsData,
+        selectInputOnFocus: true,
+        value: inputValue,
+      },
+    });
+    input = wrapper.find('input');
+    jest.spyOn(wrapper.vm, 'selectInputAndTags').mockImplementation();
+    input.setValue(inputValue);
```
Comment (marinaaisa) [inline tests/unit/components/Filter/FilterInput.spec.js:235]: Minor comment: if you already set the input value when creating the wrapper, you shouldn't need to set it up here again.

**Review by marinaaisa — APPROVED** [review]
I've been testing it out and looks great, thanks for the good work, @sofiaromorales ! 👏 
I left a minor comment.

File: tests/unit/components/Filter/FilterInput.spec.js (hunk 1 above)
Comment (sofiaromorales) [reply tests/unit/components/Filter/FilterInput.spec.js]: Thanks @marinaaisa 😄 I removed that line

Pr should be ready to go now

**Comment by marinaaisa** [comment]
@swift-ci test

**Pull request merged**